vlm/src/main/scala/geotrellis/contrib/vlm/package.scala

```diff
@@ -69,6 +75,118 @@ package object vlm {
     new StreamingByteReader(rr)
   }
 
+  implicit class tokenMethods(val token: Long) extends AnyVal {
+
+    def getProjection(): Option[String] = {
+      val crs = Array.ofDim[Byte](1 << 16)
```

Comment:
There is an allocation of a huge array, is it planned to hide such allocations to the C side and to return already a UTF-8 string?

Reply:
Alot of what you see here is stub code.